stackslib/src/chainstate/stacks/db/transactions.rs

```diff
@@ -9348,6 +9359,28 @@ pub mod test {
         let mut chainstate =
             instantiate_chainstate_with_balances(false, 0x80000000, function_name!(), balances);
 
+        let mut tx_runtime_checkerror_trait_no_version = StacksTransaction::new(
```

Comment:
Can you explain the changes in this file and how they relate to order independent multisig?

Reply:
Because this method uses `process_transaction` method, which is now gated, it is important to fix autotests to match the previous behavior and separate them by `version_opt` parameter, which is also gated